docs/platforms/flutter/session-replay/index.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Set Up Session Replay
+sidebar_order: 5500
+notSupported:
+description: "Learn how to enable the Beta of Mobile Session Replay in your app."
+---
+
+<Note>
+
+Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+
+If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-dart/issues/new?template=BUG_REPORT.yml) with a link to a relevant replay in Sentry if possible.
+
+</Note>
+
+[Session Replay](/product/explore/session-replay/) helps you get to the root cause of an error or latency issue faster by providing you with a reproduction of what was happening in the user's device before, during, and after the issue. You can rewind and replay your application's state and see key user interactions, like taps, swipes, network requests, and console entries, in a single UI.
+
+By default, our Session Replay SDK masks all text content, images, and user input, giving you heightened confidence that no sensitive data will leave the device. To learn more, see [product docs](/product/explore/session-replay/).
+
+## Pre-requisites
+
+Make sure your Sentry Flutter SDK version is at least 8.9.0, which is required for Session Replay. 
+You can update your `pubspec.yaml` to the matching version:
```

Comment:
```suggestion
## Prerequisites

Sentry Flutter SDK version 8.9.0 and up. You can update your `pubspec.yaml` by running the below:
```

Reply:
> by running the below:

that doesn't work here as below is not an executable script. It's a snippet of the configuration file and they already would have the `sentry_flutter` dependency added - they just need to make sure they have the minimum version with replay available.